examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -49,78 +62,102 @@ class GalleryDrawer extends StatelessWidget {
       child: new Block(
         children: <Widget>[
           new DrawerHeader(
-            child: new Center(
-              child: new Padding(
-                padding: const EdgeInsets.all(16.0),
-                child: new Image.asset(
-                 'packages/flutter_gallery_assets/drawer_logo.png',
-                  fit: ImageFit.contain
-                )
-              )
+            decoration: new FlutterLogoDecoration(
+              margin: new EdgeInsets.fromLTRB(12.0, 12.0 + ui.window.padding.top, 12.0, 12.0),
```

Comment:
Maybe:
  new EdgeInsets.fromWindowPadding().deflate(12.0)

Assuming that we made the default value of the EdgeInsets.fromWindowPadding(padding) parameter be ui.window.padding, and added deflate and inflate EdgeInsets methods like the Rect versions.